sources/libs/efi/base.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#define EFI_SUCCESS 0
+
+#define UNUSED(x) (void)x
+
+typedef unsigned short char16;
+typedef unsigned long long EFIStatus;
+typedef void *EFIEvent;
+typedef void *EFIHandle;
+
+typedef uint8_t u8;
```

Comment:
Why using u16 u32… here ? And if you want special type for uefi maybe add a namespace

Reply:
it was easier to type when I was declaring the efi functions. :sweat: